Kinvey/KinveyTests/KinveyTestCase.swift

```diff
@@ -100,9 +156,9 @@ class KinveyTestCase: XCTestCase {
         }
         
         if let username = username {
-            User.signup(username: username, completionHandler: handler)
+            User.signup(username: username, user: user, completionHandler: handler)
```

Comment:
the method is a generic method `signUp<UserType: User>(user: UserType?)` so it can be actually any user type